### PR TITLE
Change VTO to support GPU based simulation

### PR DESCRIPTION
**Pull request by NRHuntoon**
Changed Nodpool from "compute" to A10 to support GPU based acceleration Added "tolerations" section to match other A10 based deployments

### CODE REVIEW

File: charts/virtual-try-on/values.yaml [hunk 1]
```diff
@@ -78,13 +78,13 @@ autoscaling:
...
-  nodepool: compute
+  nodepool: "A10"
 
 tolerations:
-  - key: "compute"
+  - key: "A10"
     operator: "Equal"
     value: "true"
-    effect: "NoSchedule"
+    effect: "NoSchedule"e
```
Comment (edalford11) [inline charts/virtual-try-on/values.yaml:87]: typo

**Review by edalford11 — CHANGES_REQUESTED** [review]
You'll also need to bump the chart version in Chart.yaml. From there, you can bump the chart version in the kubernetes-deployments repo.

File: charts/virtual-try-on/values.yaml (hunk 1 above)
Comment (NRHuntoon) [reply charts/virtual-try-on/values.yaml:87]: I just bumped the version in the helm chart

File: charts/virtual-try-on/Chart.yaml [hunk 2]
```diff
@@ -15,9 +15,9 @@ type: application
...
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.1
+version: 0.1.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 1.16.0
+appVersion: 1.17.0
```
Comment (edalford11) [inline charts/virtual-try-on/Chart.yaml:23]: This doesn't do anything FYI but we can leave the change.

Comment (NRHuntoon) [reply charts/virtual-try-on/Chart.yaml:23]: Roger that. Just read the description and thought it should change

**Review by edalford11 — APPROVED** [review]

**Pull request merged**